automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/library/DataLibraryPage.scala

```diff
@@ -28,6 +29,9 @@ class DataLibraryPage(implicit webDriver: WebDriver) extends BaseFireCloudPage
   private val tags = Tags("tags")
   private val consentCodes = Tags("consent-codes")
 
+  private val tagsSearchField = SearchField(CSSQuery("[data-test-id='library-tags-select'] ~ * input.select2-search__field"))
```

Comment:
Can you explain the ` ~ *` notation and what it does here? (I'm not too familiar with it) Seems to be related to a general sibling combinator?

Reply:
The general sibling combinator (~) separates two selectors and matches the second element only if it follows the first element (not necessarily immediately), and both are children of the same parent element.

Reply:
Would it be possible to use another data-test-id instead of "input.select2-search__field"?

Reply:
No. We'd have to re-create and modify select2's internal template.